src/contractTest/java/uk/gov/hmcts/reform/cwrdapi/StaffReferenceDataProviderTest.java

```diff
@@ -176,6 +180,16 @@ private CaseWorkerProfile getCaseWorkerProfile(String caseWorkerId) {
                 new UserType(1L, "HMCTS"), false);
     }
 
+    @State({"A list of all staff reference data role-type"})
+    public void retrieveJobTitles() throws JsonProcessingException {
+        List<RoleType> roleTypeList = new ArrayList<>();
+        roleTypeList.add(new RoleType(1L, "Role Description 1"));
+        roleTypeList.add(new RoleType(2L, "Role Description 2"));
+        roleTypeList.add(new RoleType(3L, "Role Description 3"));
+        when(roleTypeRepository.findAll())
+                .thenReturn(roleTypeList);
+    }
+
 }
 
 
```

Comment:
have you missed integration tests ? i can't find.
add Integration tests for 
1. add test case for 200 with data and verify the count and their content
2. add test case for 200 empty data and verify the count and empty array

Reply:
@rajkatla-hmcts As it is new API we are not integrating to any other services. i didn't wrote the Test's. As these 200 cases are covered in functional. Correct me if i am wrong